index.html

```diff
@@ -4975,6 +4975,646 @@ <h2 id="AudioWorkerParamDescriptor">
+                by the Worker specification.
+              </p>
+              <dl class="parameters">
+                <dt>
+                  any message
+                </dt>
+                <dd></dd>
+                <dt>
+                  optional sequence&lt;Transferable&gt; transfer
+                </dt>
+                <dd></dd>
+              </dl>
+            </dd>
+          </dl>
+          <section>
+            <h2 id="AudioParamDescriptor">
+              <dfn>AudioParamDescriptor</dfn>
+            </h2>
+            <p>
+              The <code>AudioParamDescriptor</code> dictionary is used to
+              specify properties for an <a><code>AudioParam</code></a> object
+              that is used in an <a><code>AudioWorkletNode</code></a>.
+            </p>
+            <dl title="dictionary AudioParamDescriptor" class="idl">
+              <dt>
+                DOMString name
+              </dt>
+              <dd>
+                Represents the name of a parameter. An NotSupportedError
+                exception MUST be thrown when a duplicated name is found when
+                registering the class definition.
+              </dd>
+              <dt>
+                float defaultValue = 0
+              </dt>
+              <dd>
+                Represents the default value of the parameter. If this value is
+                out of the range of float data type or the range defined by
+                <code>minValue</code> and <code>maxValue</code>, an
+                NotSupportedError exception MUST be thrown.
+              </dd>
+              <dt>
+                float minValue
+              </dt>
+              <dd>
+                Represents the minimum value. An NotSupportedError exception
+                MUST be thrown if this value is out of range of float data type
+                or it is greater than <code>maxValue</code>.
+              </dd>
+              <dt>
+                float maxValue
+              </dt>
+              <dd>
+                Represents the maximum value. An NotSupportedError exception
+                MUST be thrown if this value is out of range of float data type
+                or it is smaller than <code>minValue</code>.
+              </dd>
+            </dl>
+          </section>
+          <section>
+            <h2 id="AudioContextInfo">
+              <dfn>AudioContextInfo</dfn>
+            </h2>
+            <p>
+              The <code>AudioContextInfo</code> dictionary provides an
+              <a><code>AudioWorkletGlobalScope</code></a> with a view of an
+              <a><code>AudioContext</code></a>.
+            </p>
+            <dl title="dictionary AudioContextInfo" class="idl">
+              <dt>
+                double playbackTime
+              </dt>
+              <dd>
+                The starting time of the block of audio being processed in
+                response to this event. By definition this will be equal to the
```

Comment:
Where do we define 'rendering time' in the spec? What is the difference between the 'playback' and the 'rendering' time?

Also what kind of 'latency estimation' are you referring to? Is this something similar to the context time in the device time stamp? If you mean the latency of a graph, we have decided not to offer it from the implementation.